drivers/tee/optee/Kconfig

```diff
@@ -5,3 +5,10 @@ config OPTEE
 	help
 	  This implements the OP-TEE Trusted Execution Environment (TEE)
 	  driver.
+
+config OPTEE_BENCHMARK
+	bool "OP-TEE Benchmark"
```

Comment:
I suggest to add `(EXPERIMENTAL)` to the description as it is not fully a mature today.